### ISSUE TITLE
[HOLD for payment 2022-11-09] [$250] Web - Clicking on zoom photo line should zoom the photo and the large green button should be place at clicked point reported by @aneequeahmad 

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#10950]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Click on use icon to open setting page.
2. Click on any workspace > general settings > edit photo > upload photo.
3. Click on the the zoom line

## Expected Result:
Should zoom image and drag the green button to clicked point

## Actual Result:
Not zooming image and dragging the green button.

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web

**Version Number:** v1.1.99-4
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
https://user-images.githubusercontent.com/43996225/189779379-a51506f4-1646-4f9f-a79b-9b946af9e27e.mp4
**Expensify/Expensify Issue URL:** 
**Issue reported by:** @aneequeahmad 
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by varshamb** [comment]
### Proposal

We should use Component `Slider from '@react-native-community/slider'`
I will give detailed proposal if it is okay to use it.

**Comment by tgolen** [comment]
@varshamb We typically don't use external components since most things can be built pretty easily and specifically just for our use case. Could you make a proposal that is an improvement to our current `<Slider>` instead?

**Comment by laurenreidexpensify** [comment]
Can confirm that this is reproducible.

**Comment by laurenreidexpensify** [comment]
@tgolen is this good to get the exported label?

**Comment by laurenreidexpensify** [comment]
https://www.upwork.com/jobs/~0117a7dd51bf4a5121

**Comment by laurenreidexpensify** [comment]
heh why isn't this adding a C+

**Comment by thesahindia** [comment]
Maybe because we don't have the `help wanted` label here

**Comment by Abdallah-Ahmed-moh** [comment]
### Proposal

We can replace the slider wrapper `View` with this `Pressable`
```javascrept
                                <Pressable
                                    style={[styles.mh5, styles.flex1]}
                                    onLayout={initializeSliderContainer}
                                    onPress={(e) => {
                                        const newScale = ((e.nativeEvent.layerX / sliderContainerSize)
                                    * (CONST.AVATAR_CROP_MODAL.MAX_SCALE - CONST.AVATAR_CROP_MODAL.MIN_SCALE))
                                    + CONST.AVATAR_CROP_MODAL.MIN_SCALE;
                                        translateSlider.value = e.nativeEvent.layerX;
                                        scale.value = newScale;
                                    }}
                                >
                                    <Slider sliderValue={translateSlider} onGesture={panSliderGestureEventHandler} />
                                </Pressable>
```

https://user-images.githubusercontent.com/114033410/191382818-340258d4-a784-4541-b5f7-d869c5c90787.mp4

**Comment by laurenreidexpensify** [comment]
@mananjadhav bump for review pls ^^

**Comment by mananjadhav** [comment]
@Abdallah-Ahmed-moh's proposal looks good to me and it'll work.  Though my recommendation is to dry up some lines of the code. We're already doing something similar for the green button, on L200. Let's move the logic to a common function.

@tgolen All yours.

🎀 👀 🎀 
C+ reviewed

**Comment by mananjadhav** [comment]
@tgolen Bump

**Comment by Abdallah-Ahmed-moh** [comment]
Applied in upwork, PR will be ready in 2-3 days

**Comment by mananjadhav** [comment]
@Abdallah-Ahmed-moh Any updates on this for us?

**Comment by Abdallah-Ahmed-moh** [comment]
@mananjadhav Iam sorry for being late but iam working on it.

**Comment by mananjadhav** [comment]
Okay thanks for the update @Abdallah-Ahmed-moh

**Comment by mananjadhav** [comment]
@Abdallah-Ahmed-moh could you please send a daily update on the PR or the progress? It's been 2 weeks since we've assigned the PR, would appreciate if we can get where are at?

**Comment by mananjadhav** [comment]
Quick bump @Abdallah-Ahmed-moh on PR comments.

**Comment by mananjadhav** [comment]
@Abdallah-Ahmed-moh quick bump on this PR [comment](https://github.com/Expensify/App/pull/11717#issuecomment-1286106927)

**Comment by mallenexpensify** [comment]
@laurenreidexpensify co-assigning cuz I'm part of
 [Tracking] [Polish] Image-related feature request and improvements #10894

**Comment by mananjadhav** [comment]
> @Abdallah-Ahmed-moh quick bump on this PR [comment](https://github.com/Expensify/App/pull/11717#issuecomment-1286106927)

@Abdallah-Ahmed-moh Got a chance to look at this comment?

**Comment by mallenexpensify** [comment]
@Beamanator This job is in progress so I'm not putting on hold pending #10894. 
Comment if you disagree

**Comment by Beamanator** [comment]
Agreed @mallenexpensify 👍

**Comment by aneequeahmad** [comment]
@laurenreidexpensify I have applied, thanks for mentioning

**Comment by zmeditation** [comment]
I have read your issue description on upwork site

**Comment by mananjadhav** [comment]
Applied @laurenreidexpensify

**Comment by laurenreidexpensify** [comment]
Still waiting on @Abdallah-Ahmed-moh to apply in Upwork, everyone else has been paid

**Comment by Abdallah-Ahmed-moh** [comment]
Ok i have apply on upwork on the new post..

**Comment by mananjadhav** [comment]
@laurenreidexpensify Quick bump on the above comment, so that we can get this one closed out.

**Comment by laurenreidexpensify** [comment]
@Abdallah-Ahmed-moh can you please send me a link to your Upwork profile so I can confirm I am paying the correct person? Thanks

**Comment by Abdallah-Ahmed-moh** [comment]
> @Abdallah-Ahmed-moh can you please send me a link to your Upwork profile so I can confirm I am paying the correct person? Thanks

https://www.upwork.com/freelancers/~01d7c9c17e7faeaa3e 

this is my profile, sorry for being late.

**Comment by mananjadhav** [comment]
@laurenreidexpensify Can you help with the above here and closure of the issue?

**Comment by laurenreidexpensify** [comment]
@Abdallah-Ahmed-moh contract sent in Upwork, please accept so I can proceed with payment! Thanks

**Comment by Abdallah-Ahmed-moh** [comment]
@laurenreidexpensify  i accepted your offer.

**Comment by laurenreidexpensify** [comment]
Everyone has been paid 🎉 Closing
